app/controllers/users_controller.rb

```diff
@@ -1,3 +1,27 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find(params[:id])
+  rescue RecordNotFound
```

Comment:
Nếu k tìm thấy thì chổ này nó đang sử lý như nào anh

Reply:
không tìm thấy thì ActiveRecord ném ra 1 cái Exeption là RecordNotFound, bắt cái exeption này ở ApplicationController để dùng ở mọi nơi trong ứng dụng
cách viết tường minh hơn rescue Application::RecordNotFound

Reply:
application bắt rồi thì chỗ này chỉ cần find thôi. xóa mấy dòng record not found đi